File: webpost/post.js
var deals = new Firebase('https://theredbrickinn-f566b.firebaseio.com/deals');

function getData(){
  var deal = document.getElementById("deal").value.trim();
  console.log(deal);
  saveToFB(deal);
};

function saveToFB(dealString){
  deals.push({
      deal: dealString,
      timestamp: Date.now()
  });
  document.getElementById("push").innerHTML = "Deal Posted: " + dealString;
};


function del(){
  if(confirm("Are you sure?")){
    deals.remove();
    document.getElementById("del").innerHTML = "Database Deleted";
  }
}

File: pythonscripts/post.py
import time
from datetime import datetime
from firebase import firebase
firebase = firebase.FirebaseApplication('https://theredbrickinn-f566b.firebaseio.com/', None)
# input = raw_input("Enter your deal!\n")
for x in range(1):
    firebase.post("/deals/", {'deal':"check" + str(x),'timestamp':datetime.now()})
